extensions/app-search/config/app-search.yml

```diff
@@ -0,0 +1,165 @@
+#
+# Web application listen_host and listen_port.
+# Your application will run on this host and port.
+#
+# * app_search.listen_host: Must be a valid IPv4 or IPv6 address.
+# * app_search.listen_port: Must be a valid port number (1-65535).
+#
+#app_search.listen_host: 127.0.0.1
+#app_search.listen_port: 3002
+#
+# ------------------------------ Authentication -------------------------------
+#
+# The origin of authenticated App Search users.
+# Options are standard, elasticsearch-native, and elasticsearch-saml.
+#
+# Docs: https://swiftype.com/documentation/app-search/self-managed/security
+#
+# * standard: Users are created within the App Search dashboard.
+# * elasticsearch-native: Users are managed via the Elasticsearch native realm.
+# * elasticsearch-saml: Users are managed via the Elasticsearch SAML realm.
+#
+#app_search.auth.source: standard
+#
+# (SAML only) Name of the realm within the Elasticsearch realm chain.
+#
+#app_search.auth.name:
+#
+# ---------------------------------- Workers ----------------------------------
+#
+# Configure the number of worker threads.
+#
+#worker.threads: 4
+#
+# ----------------------------------- APIs ------------------------------------
+#
+# Set to true hide product version information from API responses.
+#
+#hide_version_info: false
+#
+# ---------------------------------- Mailer -----------------------------------
+#
+# Connect App Search to a mailer.
+# Docs:
+#   https://swiftype.com/documentation/app-search/self-managed/configuration
+#
+#email.account.enabled: false
+#email.account.smtp.auth: plain
+#email.account.smtp.starttls.enable: false
+#email.account.smtp.host: 127.0.0.1
+#email.account.smtp.port: 25
+#email.account.smtp.user:
+#email.account.smtp.password:
+#email.account.email_defaults.from:
+#
+# ---------------------------------- Logging ----------------------------------
+#
+# Choose your log export path.
+#
+#log_directory: log
+#
+# Log level can be: debug, info, warn, error, fatal, or unknown.
+#
+#log_level: info
+#
+# Log format can be: default, json
+#
+#log_format: default
+#
+# Choose your Filebeat logs export path.
+#
+#filebeat_log_directory: log
+#
+# Enable logging app logs to stdout (enabled by default).
+#
+#enable_stdout_app_logging: true
+#
+# Enable ECS-style API and Analytics Logs.
+#
+#enable_ecs_logs: false
+#
+# ---------------------------------- TLS/SSL ----------------------------------
+#
+# Configure TLS/SSL encryption.
+#
+#app_search.ssl.enabled: false
+#app_search.ssl.keystore.path:
+#app_search.ssl.keystore.password:
+#app_search.ssl.keystore.key_password:
+#app_search.ssl.redirect_http_from_port:
+#
+# ---------------------------------- Session ----------------------------------
+#
+# Set key to persist user sessions through process restarts.
+#
+#secret_session_key:
+#
+# ----------------------------- Diagnostics report ----------------------------
+#
+# Path where diagnostic reports will be generated.
+#
+#diagnostic_report_directory: diagnostics
+#
```

Comment:
If this block of commented options was copied from somewhere, could we just point users to the place they could find it instead? (e.g. the App Search repo, or just the official docs) Otherwise some things will change in new versions and become irrelevant.